Dockerfile

```diff
@@ -112,5 +123,15 @@ ENV PATH /usr/src/app/bin:$PATH
 EXPOSE 5000
 VOLUME /workspace
 
+RUN gcloud auth configure-docker
+
+RUN mkdir -p ~/.config/gcloud/configurations/ \
+    && echo [core] > ~/.config/gcloud/configurations/config_default \
+    && echo project = internal-sentry >> ~/.config/gcloud/configurations/config_default \
+    && echo >> ~/.config/gcloud/configurations/config_default \
+    && echo [compute] >> ~/.config/gcloud/configurations/config_default \
+    && echo zone = us-central1-b >> ~/.config/gcloud/configurations/config_default \
+    && echo region = us-central1 >> ~/.config/gcloud/configurations/config_default
```

Comment:
So, this is a bit weird since it's pinning to only work for us and literally using our `internal-sentry` project name.

If we do this in the entrypoint.sh, we can take advantage of laying down runtime configuration from environment variables. Or expect this to be mounted in from the host. Basically, we can't provide this configuration hardcoded in the image.